build.gradle

```diff
@@ -273,6 +273,7 @@ task python37PostCommit() {
 
 task python38PostCommit() {
   dependsOn ":sdks:python:test-suites:portable:py38:crossLanguagePythonJavaKafkaIOFlink"
+  dependsOn ":sdks:python:test-suites:portable:py38:crossLanguagePythonJavaJdbcIO"
```

Comment:
Probably add to other post-commits as well ?

Reply:
I'm ok with that. The reason why I chose py38 only was that KafkaIO cross-language tests are run on py38 only.

Reply:
Some tests will run only in high-priority python versions and that's ok. @lazylynx  is working on making the workflow easy to configure in https://github.com/apache/beam/pull/12239.

Reply:
@tvalentyn should I remove this from other python versions then?